test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/token/HapiTokenAssociate.java

```diff
@@ -78,17 +84,24 @@ protected HapiTokenAssociate self() {
 	@Override
 	protected long feeFor(HapiApiSpec spec, Transaction txn, int numPayerKeys) throws Throwable {
 		try {
-			final long expiry = lookupExpiry(spec);
-			FeeCalculator.ActivityMetrics metricsCalc = (_txn, svo) -> {
-				var estimate = TokenAssociateUsage.newEstimate(_txn, suFrom(svo))
-						.givenCurrentExpiry(expiry);
-				return estimate.get();
-			};
+			expiry = lookupExpiry(spec);
 			return spec.fees().forActivityBasedOp(
-					HederaFunctionality.TokenAssociateToAccount, metricsCalc, txn, numPayerKeys);
+					HederaFunctionality.TokenDissociateFromAccount, this::usageEstimate, txn, numPayerKeys);
```

Comment:
shouldn't this be `HederaFunctionality.TokenAssociateToAccount`

Reply:
Yes, fixed.